packages/video_player/video_player/ios/Classes/messages.m

```diff
@@ -0,0 +1,307 @@
+// Autogenerated from Dartle.
+#import "messages.h"
+#import <Flutter/Flutter.h>
+
+#if !__has_feature(objc_arc)
+#error File requires ARC to be enabled.
+#endif
+
+static NSDictionary* wrapResult(NSDictionary *result, FlutterError *error) {
```

Comment:
uber ultra nit: be consistent on where in the file you add this static between java/objc